src/pages/Cats.js

```diff
@@ -0,0 +1,21 @@
+import { useDispatch } from 'react-redux';
+import { checkStatus } from '../redux/Categories/Categories';
+
+const Cats = () => {
+  const dispatch = useDispatch();
+
+  return (
+    <div className="cats">
+      <div
+        className="cat-div"
+        onClick={() => {
+          dispatch(checkStatus());
+        }}
+      >
+        Under Construction
+      </div>
+    </div>
+  );
+};
+
+export default Cats;
```

Comment:
- [ ] Your category component is not well implemented. The "Under Construction" text should not be on the page when the component is loaded. Rather there should be a "Check status" button which when clicked will dispatch an action to the store which then displays the "Under Construction" text.

  ![image](https://user-images.githubusercontent.com/81468675/203796927-2be9e9cd-3f06-4d93-a7c7-41aee6628304.png)

Reply:
@julie-ify ok now it works perfectly